lib/views/item/item-detail.js

```diff
@@ -95,15 +114,15 @@ class ItemDetail extends React.Component {
     );
   }
 
-  renderUrlBar({ url }) {
+  renderUrlBar({ url } = {}) {
```

Comment:
This default values are required for the smoke test we have.
Otherwise we'd have to mock a complex props. This way has the additional advantage of improved robustness if some expected values are missing for some reason.